app.coffee

```diff
@@ -13,6 +13,26 @@ rate_limit = require('./rate_limit.coffee')
 app = express().http().io()
 announcementHandler = require('./announcements.coffee')(app.io, slack)
 userInfoHandler = require('./userinfo.coffee')(app.io, slack)
+showdown = require('showdown')
+showdownHtmlEscape = require('showdown-htmlescape')
+sanitizeHtml = require('sanitize-html')
+
+converter = new showdown.Converter
+  extensions: [showdownHtmlEscape]
+  simplifiedAutoLink: true
+  openLinksInNewWindow: true
+  strikethrough: true
+
+sanitizerOptions = 
+  allowedTags: ['b', 'i', 'em', 'strong', 'a', 'strike', 'del', 'code', 'img']
+  allowedAttributes:
+    'a': ['href', 'target']
+    'img': ['src']
```

Comment:
I'd advice against allowing img tags. Has potential for abuse. Lets an attacker gets everyone's IP address

Reply:
After seeing this [what-could-an-img-src-xss-do?](https://security.stackexchange.com/questions/135513/what-could-an-img-src-xss-do), I definitely agree!